apps/src/code-studio/pd/workshop_dashboard/attendance/session_attendance.jsx

```diff
@@ -49,6 +49,7 @@ const SessionAttendance = React.createClass({
     this.showSectionMembership = !this.shouldUseNewAttendance && this.props.accountRequiredForAttendance;
     this.showPuzzlesCompleted = this.shouldUseNewAttendance && this.isCSF;
     this.isAdmin = window.dashboard.workshop.permission === "workshop_admin";
+    this.isPlp = window.dashboard.workshop.permission.indexOf('plp') >= 0;
```

Comment:
this seems a bit tenuous. Do we really not have a better way of determining that?